docs/root/api-docs/xds_protocol.rst

```diff
@@ -500,14 +500,17 @@ Deleting Resources
 
 In the incremental protocol variants, the server signals the client that a resource should be
 deleted via the :ref:`removed_resources <envoy_v3_api_field_service.discovery.v3.DeltaDiscoveryResponse.removed_resources>`
-field of the response. This tells the client to remove the resource from its local cache.
+field of the response for wildcard resource types like
```

Comment:
How about add a section named "wildcard resource" and remove `like` here?

Reply:
added "wildcard resource" paragraph